test/m1_facilitator/Config/fromFile.test.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs-extra';
+import yaml from 'js-yaml';
+import Config, {
+  DBConfig, Avatar, Chain, Metachain,
+} from '../../../src/m1_facilitator/Manifest/Manifest';
+import assert from '../../test_utils/assert';
+
+interface AccountDetail {
+  keystore_path: string;
+  keystore_password_path: string;
+}
+
+describe('Config.fromFile()', (): void => {
+  it('should return correct facilitator config object', async (): Promise<void> => {
+    const manifestFilePath = 'testdata/m1_facilitator/facilitator_manifest.yml';
+    const inputYamlConfig = yaml.safeLoad(fs.readFileSync(manifestFilePath, 'utf8'));
+    const manifest = Config.fromFile(manifestFilePath);
+
+    assert.strictEqual(
```

Comment:
is it possible to do deepEquals 'inputYamlConfig' and 'manifest'?

Reply:
No!
The values of manifest object are different.